nilearn/datasets.py

```diff
@@ -26,6 +25,17 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    if uptime.uptime() is None:
+        raise Exception("uptime is not supported by this OS.")
+    _get_time = uptime.uptime
```

Comment:
> I am somewhat agnostic, but I would suggest that the data fetching part
> of this package could really be an attractive aspect of it.

Maybe, but we are speculating on the future. Who knows how the resources
to maintain nilearn are going to evolve, and how the demand is going to
evolve. Right now, we are a small team, and I am afraid that the
complexity of the download code is starting to become disproportionate to
its usefulness. Importantly, the priority of nilearn is to be useful to
neuroimaging and we need to invest best our efforts to find the simplest
way to achieve this goal.